coalib/output/ConsoleInteraction.py

```diff
@@ -426,6 +426,29 @@ def print_affected_lines(console_printer, file_dict, section, sourcerange):
                         sourcerange)
 
 
+def join_names(values):
+    """
+    Produces a string by concatenating the items in ``values`` with
+    commas and an "and" for the last element. Example:
```

Comment:
this function looks good, but i dont really like the docstring.

maybe something like "Produces a string by concatenating the items in values with commas, besides the last element, which is concatenated with an ``and``".